File: sea-shell-runnables/sea-shell-client/src/main/java/org/jesperancinha/shell/webflux/client/data/SeaShellTopDto.java
package org.jesperancinha.shell.webflux.client.data;

import lombok.*;

@Builder
@Getter
@Setter
@EqualsAndHashCode
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class SeaShellTopDto {
    private String type;
    private String color;
    private String size;
}

File: sea-shell-service-spring-web-flux/src/main/kotlin/org/jesperancinha/shell/webflux/data/SeaShellTopDto.kt
package org.jesperancinha.shell.webflux.data

import lombok.*

@Builder
@Getter
@Setter
@EqualsAndHashCode
@ToString
@AllArgsConstructor
@NoArgsConstructor
class SeaShellTopDto {
    private val type: String? = null
    private val color: String? = null
    private val size: String? = null
}